soa-suite/README.md

```diff
@@ -0,0 +1,320 @@
+
+This deployment of Oracle SOA Suite makes use of the Oracle SOA Suite Helm Chart based on the [fmw-kubernetes](https://github.com/oracle/fmw-kubernetes) release.
+
+The Helm chart is provided as an example and is not currently officially supported by Oracle. Refer to the [fmw-kubernetes](https://github.com/oracle/fmw-kubernetes) release for the officially supported deployment.
+
+## Caveats
+
+Although this release follows the same flow as the [fmw-kubernetes](https://github.com/oracle/fmw-kubernetes) release, only the Traefik ingress controller is currently supported.
+
+
+## 1. Mandatory Prerequisites
+
+To deploy this chart, you will need:
+
+- An Oracle Container Registry account
+- Acknowledge terms of use for the Oracle SOA Suite Docker image on the Oracle Container Registry.
+- Acknowledge terms of use for the Oracle Database Docker image on the Oracle Container Registry to deploy the test Oracle database in your cluster (NOT for production).
+- A Network File Server (NFS) and mount point accessible from within your cluster.
+
+### 1.1 Oracle Container Registry
+
+You will need an Oracle Container Registry account here: [https://container-registry.oracle.com/pls/apex/f?p=113:10:18395320492442:::RP,10::](https://container-registry.oracle.com/pls/apex/f?p=113:10:18395320492442:::RP,10::)
+
+Log in with your Oracle account credentials or create a new account, and agree to the terms of use for the images you need to use:
+
+### 1.2 Oracle SOA Suite Docker Image
+
+The chart uses the Oracle SOA Suite Docker image from the Oracle Container Registry. This is a mandatory requirement.
+
+You must accept the terms of use for this image before using the chart, or it will fail to pull the image from registry.
+
+- At [https://container-registry.oracle.com/pls/apex/f?p=113:10:18395320492442:::RP,10::](https://container-registry.oracle.com/pls/apex/f?p=113:10:18395320492442:::RP,10::), search for 'SOA'.
+- Click **soasuite**.
+- Click to accept the License terms and condition on the right.
+- Fill in your information (if you haven't already).
+- Accept the License.
+
+### 1.3 Oracle Database Docker Image
+
+You may provision the database supporting the Oracle SOA suite domain schemas separately, and point the chart to it by providing the database url. The database must be accessible from the Kubernetes cluster. This is the recommended way to deploy this chart.
+
+If you intend on deploying the database within the kubernetes cluster (optional; not for production), you must agree to the terms of the Oracle database Docker image:
+
+- Search for Database.
+- Click **Enterprise**.
+- Click to accept the License terms and condition on the right.
+- Fill in your information (if you haven't already).
+- Accept the License.
+
+Note that the deployement in cluster is for testing purpose only and not for production.
+
+## 2. Other Prerequisites
+
+To easily deploy the prerequisites through a Terrafrom script, check the related repository here: [fixme](fixme)
```

Comment:
Please fix the `fixme`.

Reply:
fixed the fixme but this is a catch 22: url won't work until the chart is published and i can publish the related terraform.